definitions/infra-kubernetespod/definition.yml

```diff
@@ -17,6 +17,9 @@ synthesis:
     k8s.namespaceName:
     k8s.podName:
     k8s.nodeName:
+    k8s.deploymentName:
```

Comment:
is this for k8sPod or k8sDeployment?

Reply:
For both. We also include `k8s.deploymentName` in pod metrics, in case the pods were created by a Deployment.

Reply:
thanks! (just curious).